KeyActions/src/main/java/org/vaadin/artur/client/KeyActionConnector.java

```diff
@@ -44,12 +42,6 @@ protected void triggerEvent(final KeyDownEvent event) {
 		Scheduler.get().scheduleDeferred(new ScheduledCommand() {
 			@Override
 			public void execute() {
-				if (getParent() instanceof BeforeShortcutActionListener) {
```

Comment:
Do you mean call flush() before the RPC call? Please see the modified file if that's what you meant.